### PR TITLE
added/fixed translations + regexes for shb dungeons

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js [hunk 1]
```diff
@@ -209,6 +213,11 @@
         'Acrid Stream': 'Ätzende Strömung',
         '2,000-mina swipe': '2000 Mina-Hiebe',
         '2,000-mina swing': '2000 Mina-Schwung',
+        'Swing/Swipe/Cyclone': '',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js]: ?

File: ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js [hunk 2]
```diff
@@ -147,7 +147,7 @@
         'trench phuabo': 'Tiefsee-Phuabo',
         'trench danbania': 'Tiefsee-Danbania',
         'trench anemone': 'Tiefsee-Anemone',
-        'sinister bubble': 'Finster(?:e|er|es|en) Blase',
+        'Sinister Bubble': 'Finster(?:e|er|es|en) Blase',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js]: I think these are all case insensitive.  It's probably not worth changing the case on any keys.  I mean, this is fine as-is, but just saying for the future.

Comment (Akurosia) [reply ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js]: i just change this as they were reported by the python script for checking missing sync/text

File: ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js (hunk 1 above)
Comment (Akurosia) [reply ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js]: seams i missed this one after translating the rest...

**Pull request merged**